server/src-lib/Hasura/GraphQL/Validate/Field.hs

```diff
@@ -54,6 +54,7 @@ data Field
   , _fType      :: !G.NamedType
   , _fArguments :: !ArgsMap
   , _fSelSet    :: !SelSet
+  , _fSource    :: !TypeLoc
```

Comment:
Field will have a `TypeLoc` which is either `Hasura` or `Remote`

Reply:
This should perhaps change to `Hasura`, `Remote` or `RemoteRel` going fwd in RJ branch.